lib/frozen_record/backends/yaml.rb

```diff
@@ -0,0 +1,27 @@
+module FrozenRecord
+  module Backends
+    class Yaml
```

Comment:
Don't use a class if it's never instantiated. In this case a self extended module will do:

```ruby
module Yaml
  extend self

  def blah
```

Reply:
You are right! I thought this would become a proper class with instance methods eventually, but turns out it didn't. 😄